test/unit/presenters/api/applications_new_presenter_test.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Api::ApplicationsNewPresenterTest < ActiveSupport::TestCase
+
+  Presenter = Api::ApplicationsNewPresenter
+
+  def setup
+    @provider = FactoryBot.create(:provider_account)
+    @service = FactoryBot.create(:service)
+    @user = FactoryBot.create(:simple_admin)
+  end
+
+  attr_reader :provider, :service, :user, :cinstance
+
+  test 'new_application_form_data' do
+    presenter = Presenter.new(provider: provider, service: service, user: user)
+    form_data = presenter.new_application_form_data
+
+    assert form_data.key? :"create-application-path"
+    assert form_data.key? :"create-application-plan-path"
+    assert form_data.key? :"service-subscriptions-path"
+    assert form_data.key? :"service-plans-allowed"
+    assert form_data.key? :"defined-fields"
+    assert form_data.key? :"most-recently-created-buyers"
+    assert form_data.key? :"buyers-count"
+    assert form_data.key? :product
+
+    assert_not form_data.key? :buyer
+    assert_not form_data.key? :"most-recently-updated-products"
+    assert_not form_data.key? :"products-count"
+    assert_not form_data.key? :errors
```

Comment:
1. `refute` is preferred to `assert_not`
2. `assert_does_not_contain` is more readable ;)

```ruby
%i[buyer most-recently-updated-products products-count errors].each do |key|
  assert_does_not_contain form_data.keys, key
end
```

Reply:
No `refute_same_elements` then?